web/packages/teleterm/src/deepLinks.test.ts

```diff
@@ -180,8 +221,30 @@ describe('parseDeepLink followed by makeDeepLinkWithSafeInput gives the same res
     const { url } = parseResult as DeepLinkParseResultSuccess;
     const deepLink = makeDeepLinkWithSafeInput({
       proxyHost: url.host,
-      path: url.pathname.substring(1) as Path, // Remove the leading slash.
+      path: url.pathname,
+      username: url.username,
+      searchParams: {},
+    });
+    expect(deepLink).toEqual(input);
+  });
+});
+
+describe('"parseDeepLink followed by makeDeepLinkWithSafeInput gives the same result"', () => {
```

Comment:
It looks like we ended up with two `describe` blocks with the same message but slightly different texts, could you fix that in the other PR?

Reply:
Oh oops. Yes will push to the other PR